### ISSUE TITLE
Call not being rejected in Oreo

### ISSUE THREAD

**Issue by TheDcoder** [kaliturin/BlackList#12]
I have just installed this on my mum's older phone and it is working very well.

After getting impressed I installed it too but it failed my simple test, I added an contact and the app didn't reject the call, it did show the notification about the call being blocked.

I am using Android Oreo and I have given it all of the requested permission on the first launch.

**Comment by rocode** [comment]
Confirmed not working in Android 9 (Pie) either.

**Comment by dhazel** [comment]
Happening for me too.

Google Pixel 2, Android 9

**Comment by heetparkhiya** [comment]
you need to replace the default dialer to block the call

**Comment by parcelcat** [comment]
Version 1.2.12 of the app is also not blocking any calls on Android 9 Pie. The app displays a notification appears saying "Private number: call is blocked", but the call is not blocked and the number is not private.

**Comment by betsythefc** [comment]
@heetparkhiya I have Simple Contacts as my default "Phone app", but the problem persists.
However, I have noticed that when I get or place a call, the actual in-call interface is the stock phone app. Is there somewhere I can change my actual dialing app?

I have a Motorola G6 on Sprint.

**Comment by heetparkhiya** [comment]
@betsythefc yes you can change the dialer but changing the default with your custom is too unacceptable because your dialer should have minimum working funcationality like system app and in some case system will also not allow to change default dialer
